lib/link_checker/uri_checker/http_checker.rb

```diff
@@ -133,6 +133,7 @@ def check_google_safebrowsing
     def make_request(method)
       begin
         response = run_connection_request(method)
+        return nil unless response
```

Comment:
this code will probably not get executed, as it would throw an exception and you'd get hoisted into the rescue blocks.

Side note: you also don't need to add nil for this scenario, an empty return implicitly returns nil